core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -409,6 +409,52 @@ class Router(formatOps: FormatOps) {
                 .withIndent(2, expire, Left)
             )
         }
+
+      // Parameter opening for one parameter group. This format works
+      // on a group-by-group basis.
+      case FormatToken(open @ LeftParen(), right, between)
+          if style.verticalMultiline && isDefnSite(leftOwner) =>
+        val close = matchingParentheses(hash(open))
+        val indentParam = Num(4)
+        val indentSep = Num(2)
+        if (isTuple(leftOwner)) {
```

Comment:
This is actually not needed, left-over from when I was copy-and-pasting. Sorry for the noise.